src/components/Aframe/my-loader.js

```diff
@@ -68,27 +68,26 @@ AFRAME.registerComponent("myloader", {
     this.onSelectStart = this.onSelectStart.bind(this);
     this.onClearCollide = this.onClearCollide.bind(this);
     this.loadModel = this.loadModel.bind(this);
-    this.updateTransfertexture = this.updateTransfertexture.bind(this);
+    this.updateTransferTexture = this.updateTransferTexture.bind(this);
     this.updateColorMapping = this.updateColorMapping.bind(this);
     this.debugScene = this.debugScene.bind(this);
 
+    this.updateOpacityData = this.updateOpacityData.bind(this);
+    this.colorMapNeedsUpdate = false;
+    this.currentColorMap = "";
```

Comment:
If you're using `this.currentColorMap` in the rest of the code I would initialize it as `this.currentColorMap = this.data.colorMap`

Reply:
done !